### ISSUE TITLE
`system.asd` didn't get properly expanded

### ISSUE THREAD

**Issue by aadcg** [atlas-engineer/ntemplate#4]
While testing it I noticed that the `.asd` file got expanded into

```lisp
(defsystem "foo/tests"
  :depends-on ((#| TMPL_VAR name |#) lisp-unit2)
```

but the expected output was

```lisp
(defsystem "foo/tests"
  :depends-on (foo lisp-unit2)
```

**Comment by aartaka** [comment]
Should be fixed with b1476ba6abc9dd8088e5ff971f09d085bb67f470.

I've noticed and fixed it long ago on my local checkout, but forgot to push :sweat_smile:

**Comment by aadcg** [comment]
Thanks!
